admiral/pkg/controller/admiral/dependency_test.go

```diff
@@ -20,3 +23,76 @@ func TestNewDependencyController(t *testing.T) {
 		t.Errorf("Dependency controller should never be nil without an error thrown")
 	}
 }
+
+func TestGetDependencies(t *testing.T) {
+	stop := make(chan struct{})
+	handler := test.MockDependencyHandler{}
+
+	dependencyController, err := NewDependencyController(stop, &handler, "../../test/resources/admins@fake-cluster.k8s.local", "ns", time.Duration(1000))
+
+	if err != nil {
+		t.Errorf("Unexpected err %v", err)
+	}
+
+	if dependencyController == nil {
+		t.Errorf("Dependency controller should never be nil without an error thrown")
+	}
+
+	deps, _  := dependencyController.GetDependencies()
```

Comment:
Can you add a case to this test where there are dependencies to get? Otherwise we're not really testing the functionality, as 

```
func (d *DependencyController) GetDependencies() ([]v1.Dependency, error) {
  return nil, nil
} 
```
also passes this test.